text/0027-co-locate-all-rfcs.md

```diff
@@ -0,0 +1,72 @@
+# Co-locate All Paketo RFCs
+
+## Summary
+
+All individual repository level RFCs that are in the Paketo organization should
+be centralized into the top level RFCS repository.
+
+## Motivation
+
+As it stands currently there are RFCs scattered across various repositories
+throughout the Paketo organization. These RFCs pertain typically to the
+implementation and functionality of either entire language families or even on
+down to implementation buildpack specific behavior. While it is good to have a
+record of technical decisions made for the project and its components, it is
+sometimes hard to search for relative RFCs for a given language family or
+feature. By co-locating all of the RFCs for the project into one repository we
+will collect all of the decision making conversation into one place making it
+easier to access.
+
+## Implementation
+
+Each project team will be given a directory inside of the `rfcs` repository,
+which would be named after the project team. A series of directories with the
+same names as the repositories that fall under that project teams control will
+be created inside of the project team's sub-directory.
+```
+rfcs
+├── assets
+│   └── ...
+├── LICENSE
+├── NOTICE
+├── README.md
+└── text
+    ├── 0000-template.md
+    ├── ...
```

Comment:
I think this is sort of implied, but is it your intent that there are still RFCs outside of project subdirectories? These would be RFCs that need to be voted on by the whole project and not just a subteam, I'm guessing.

Reply:
The RFCs that are outside of the project subdirectories would be Paketo level RFCs that need to be approved by the steering committee.